src/main/java/com/google/sps/servlets/ColumnsFromTablesServlet.java

```diff
@@ -60,7 +60,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
           for(int i = 0; i < listOfTables.length; i++){
               if(i != listOfTables.length-1){
                 constant.SELECTED_TABLES = "\'" + listOfTables[i] + "\'";
-                constant.QUERY = constant.QUERY + constant.SELECTED_TABLES + constant.COMMA;
+                constant.QUERY = constant.QUERY + constant.SELECTED_TABLES + ", ";
```

Comment:
(A pending comment from the last PR, that I didn't get to send it out. I don't feel strong about this, if you keep the query this way for now, that is fine. )

Many of the atom constants can be avoided by formatting a string rather than concatenating strings. For example:

SCHEMA_INFO_SQL = "SELECT column_name, spanner_type, is_nullable FROM information_schema.columns WHERE table_name = '%s'"
And we can create the query with String.format(SCHEMA_INFO_SQL, table).

I think this makes it easier for readers to get what the queries are doing.

Reply:
I like this idea a lot. I will make a note of this and try to implement it in my other servlets in a later PR.